src/containers/ForgotPassword/index.js

```diff
@@ -0,0 +1,146 @@
+// This is a smart component which is aware of redux
+import { bindActionCreators } from 'redux';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import InputField from '../../components/InputField';
+import Errors from '../../components/Errors';
+import validateInput from '../../utils/validateInput';
+import { forgotPasswordAction } from './actions';
+
+
+class ForgotPassword extends Component {
+  static defaultProps = {
+    forgotPass: {},
+  }
+
+  state = {
+    email: '',
+    validation: {},
+  };
+
+  handleChange = (e) => {
+    this.setState({
+      [e.target.name]: e.target.value,
+      validation: { [e.target.name]: validateInput(e.target.name, e.target.value) },
+    });
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    const { sendLink } = this.props;
+    sendLink({ email: this.state.email });
+  };
+
+  renderInput = (name, failure, errors, value) => (
+    <InputField
+      name={`${name}`}
+      label={`Enter your ${name}`}
+      type={`${name}`}
+      value={value}
+      onChange={this.handleChange}
+      failure={failure}
+      errors={errors}
+    />
+  )
+
+  render() {
+    const {
+      errors, failure, isFetching, success,
+    } = this.props.forgotPass;
+    let output;
+
+    if (success) {
+      output = (
+        <div className="row">
```

Comment:
Can you make this one component, please? this is a lot of code being repeated

Reply:
I'll work on it.